### PR TITLE
Dev card clipping (JS function)

### PR DESCRIPTION
**Pull request by ivokwee**
Xavier solution for dropdown clipping problem using a JS function. The JS code is now coded as function to prevent too much code attaching to each card/plotmodule.  Previous solutions still crashed some browsers with out of memory error. Please test and report.

### CODE REVIEW

**Review by ESCRI11 — APPROVED** [review]

File: components/ui/ui-DropDownMenu.R [hunk 1]
```diff
@@ -183,7 +77,6 @@ actionMenu <- function(..., size = "default", status = "default", icon = NULL, m
         },
         error = function(w) {}
       )
-    ),
-    dropdown.assets(id)
+    )
```
Comment (cpsievert) [inline components/ui/ui-DropDownMenu.R:80]: Are we sure this isn't also needed for `ActionMenu()`?

**Review by cpsievert — APPROVED** [review]
LGTM (other than the one question)

File: components/ui/ui-DropDownMenu.R (hunk 1 above)
Comment (ESCRI11) [reply components/ui/ui-DropDownMenu.R]: Good catch. Just fixed it

**Comment by phisanti** [comment]
I have just tested this new version, and I can confirm that the "Out of memory error" does not occur. However, I see that there are 3 versions of the JS code: `dropdown.jsCode.SAVE2`, `dropdown.jsCode.SAVE1` and `dropdown.jsCode`. Before merging into the main, these should be clean so we don't clutter the main project. Ideally, we should consider moving the `www` directory.

**Comment by ESCRI11** [comment]
@phisanti It is removed here https://github.com/bigomics/omicsplayground/pull/606/commits/a469f4d88a57701af51248dc379662ed82afc086

**Pull request merged**